libmozdata/patchanalysis.py

```diff
@@ -547,21 +547,32 @@ def attachmenthandler(attachments, bugid):
         'uplift_accepted': False,
         'uplift_comment': None,
         'uplift_author': None,
+        'uplift_reviewer': None,
         'landing_delta': timedelta(),
         'response_delta': timedelta(),
         'release_delta': timedelta(),
     }
     approval_flag = 'approval-mozilla-' + channel
 
-    app_flags = filter(lambda f: f['name'] == approval_flag, [flag for a in bug['attachments'] for flag in a['flags']])
+    app_flags = [
+        flag
+        for a in bug['attachments']
+        for flag in a['flags']
+        if flag['name'] == approval_flag
+    ]
     status = [flag['status'] for flag in app_flags]
+    reviewers = [flag['setter'] for flag in app_flags]
```

Comment:
Nit: let's call this `uplift_reviewers` to make it more clear that it doesn't hold the reviewers of the patch